### PR TITLE
4 All reviews

### PR DESCRIPTION
**Pull request by fedog-ops**
add header, navbar, homepage. hompage displays all reviews

### CODE REVIEW

File: src/components/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { getReviews } from "../utils/API"
+import { useEffect, useState } from "react"
+import header from './styling/header.css'
+
+const Homepage = () => {
+    
+const [reviews, setReviews] = useState([])
+
+useEffect(() => {
```
Comment (Augs0) [inline src/components/Homepage.jsx:9]: I would usually complain that the console log was left in as we want to take these out before pushing, but in this case it helped me see a huge problem here - this is currently an infinite loop! What do you think is going on in your useEffect to cause that? Please make sure to fix this before anything else!

**Review by Augs0 — CHANGES_REQUESTED** [review]
Great work Felix but just a sneaky infinite loop in here that should be fixed asap! Please fix and resubmit the same Pr to helpdesk :)

File: src/components/Homepage.jsx (hunk 1 above)
Comment (fedog-ops) [reply src/components/Homepage.jsx:9]: a [] dependency variable for the use effect

**Review by alexdevdavis — APPROVED** [review]
Nicely done, happy to approve now

**Pull request merged**